backend/mongodb/models/threads.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from mongoengine import (
+    Document,
+    EmbeddedDocument,
+)
+from mongoengine.fields import (
+    BooleanField,
+    DateTimeField,
+    EmbeddedDocumentField,
+    IntField,
+    ListField,
+    ReferenceField,
+    StringField,
+    URLField,
+)
+
+class Thread(Document):
+    meta = {"collection": "threads"}
+
+    name = StringField()
+    forum = ReferenceField("Forum")
+    pinned = BooleanField()
+    content = StringField()
+    created_at = DateTimeField(default=datetime.datetime.now)
+    creator = ReferenceField("User")
+    last_activity = DateTimeField(default=datetime.datetime.now)
```

Comment:
@parvoberoi do we need to test the `created_at` and `last_activity` field defaults? Guessing one of these needs to change since `created_at` is only assigned a value during creation but `last_activity` is updated on every `update/save`